mage_ai/frontend/components/DependencyGraph/BlockNode/index.tsx

```diff
@@ -205,7 +205,7 @@ function BlockNode({
             </Flex>
           </Flex>
 
-          <Spacing mr={HEADER_SPACING_HORIZONTAL_UNITS} />
+          <Spacing mr="15px" />
```

Comment:
Decreased to 15px (instead of 16px) to fix styling that cuts off text when screen is zoomed out (e.g. to 90% on certain screen displays).